### DISCUSSION TITLE
Student Benefits Problem

### DISCUSSION THREAD

**Discussion by JoeAfaridGhost** [community/community#35517] [GitHub Education]
I wanna upload proof of my academic status, but there is only one way to take photos of it. I can't submit the jpg form file. As an international student, I only have an e-mail to prove I have enrolled in my course. Because of the pandemic, I have no physical ID card.

**Comment by MitkoVtori** [comment]
I don't know what device you're using, but you can open the jpg file and save a copy of it as png. Also you can take a screenshot of your jpg picture and save the screenshot as png or else.
Windows copy example:
![Screenshot 2022-10-09 100347](https://user-images.githubusercontent.com/112943652/194742949-f31318b1-364c-4b28-9a14-d31b35c2245a.png)
![Screenshot 2022-10-09 100504](https://user-images.githubusercontent.com/112943652/194742952-a4978c20-8d44-47c2-9919-e7b9f6bd7b31.png)
![Screenshot 2022-10-09 100553](https://user-images.githubusercontent.com/112943652/194742958-103e54a0-8e30-410a-bbdf-44915dc53aaf.png)
![Screenshot 2022-10-09 100635](https://user-images.githubusercontent.com/112943652/194742963-4cd0818d-09ef-40fb-8729-13a5075eaae3.png)

It should work on any computer/laptop. Here is the screen shot example:
go to the picture(maybe to your email because you said is there) press Windows button, shift and S at once or in the order I said, and up on your screen click on screen shot, after that you can cut the picture:
![Screenshot 2022-10-09 100943](https://user-images.githubusercontent.com/112943652/194743141-7f910002-4885-4550-88ae-9bde0e9277b4.png)
If non-of this two works, try this:
https://www.lifewire.com/convert-jpg-to-png-5072559

If this help you, you can save my respond as answer so other people with the same problem can see that this question is answered.
